nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/test/java/org/nuxeo/ecm/platform/rendition/service/TestRenditionService.java

```diff
@@ -702,7 +707,7 @@ public void shouldRemoveFilesBlobsOnARendition() {
     }
 
     @Test
-    public void shouldNotRenderADocumentWithoutBlobHolder() {
+    public void shouldNotRenderANonFolderishDocumentWithoutBlobHolder() {
```

Comment:
Shouldn't be `shouldNotRenderAFolderishDocumentWithoutBlobHolder` ? (-Non)

Reply:
A Folderish without a BlobHolder is the normal condition and should be renderable. A NonFolderish without a BlobHolder should fail. This method name attempts to capture that condition. Unfortunately, the double-negatives in the name are a bit confusing. But I think the name captures the intent of the test.